pycytominer/cyto_utils/cell_locations_cmd.py

```diff
@@ -0,0 +1,5 @@
+from pycytominer.cyto_utils.cell_locations import CellLocation
+import fire
```

Comment:
do we need to add `fire` to the requirements?

Reply:
yes; will do